conversions/binary_to_decimal.c

```diff
@@ -1,24 +1,31 @@
-/**
- * Modified 07/12/2017, Kyler Smith
```

Comment:
You might want to keep this. We can create another PR to improve the documentation of this file. 🙂